cmd/zpool/zpool_main.c

```diff
@@ -7138,6 +7164,9 @@ status_callback(zpool_handle_t *zhp, void *data)
 		    cbp->cb_namewidth, "NAME", "STATE", "READ", "WRITE",
 		    "CKSUM");
 
+		if (cbp->cb_print_slow_ios)
+			(void) printf(gettext("  SLOW"));
```

Comment:
It would be easier to see that this is correctly aligned if we did `" %5s"`, like we do with the other headers, and when printing the value